rimage/transform/distorter.go

```diff
@@ -6,8 +6,6 @@ import "github.com/pkg/errors"
 type DistortionType string
 
 const (
-	// NoneDistortionType applies no distortion to an input image. Essentially an identity transform.
-	NoneDistortionType = DistortionType("no_distortion")
```

Comment:
Thanks for getting rid of this, making "no distortion" just be nil is much simpler